src/ballet/hashmap/polymur-hash.h

```diff
@@ -0,0 +1,295 @@
+/*
+    PolymurHash version 2.0
```

Comment:
Please add a section to `NOTICE` regarding this files source, and its license.

Also please note here the commit hash you took and modified for future reference

Reply:
Actually removed PolymurHash.